components/script/dom/document.rs

```diff
@@ -724,6 +728,8 @@ impl Document {
                     }
                     // Step 4.6.2
                     document.page_showing.set(true);
+                    // Step 4.6.3
```

Comment:
Please add descriptive text with the step numbers as the numbers change all the time.